lighthouse-core/report/html/renderer/logger.js

```diff
@@ -49,6 +49,11 @@ class Logger {
    */
   error(msg) {
     this.log(msg);
+
+    // Rethrow to make sure it's auditable as an error.
+    setTimeout(_ => {
+      throw new Error(msg);
```

Comment:
Why the settimeout?

Reply:
> Why the settimeout?

it's a hack that lets logger catch it so the report still acts nicely (with a toast) even if the report is invalid or whatever, but now it also async throws the error so that the problem can be picked up in the console and (more importantly) in the puppeteer `pageerror` listener we add in the test :)

Reply:
added to the comment to clarify